database/migrations/2022_03_09_001334_add_eula_to_checkout_acceptance.php

```diff
@@ -13,10 +13,12 @@ class AddEulaToCheckoutAcceptance extends Migration
      */
     public function up()
     {
-        Schema::table('checkout_acceptances', function (Blueprint $table) {
```

Comment:
This doesn't really seem related, is this deliberate?

Reply:
This was necessary because I was getting a "field already exists" error when migrations were being run